### PR TITLE
build: add common release script

### PR DESCRIPTION
**Pull request by tiagolobocastro**
The script header showcases how to use it.
This will allow other repos to simply make use of this script and add unique requirements on top of it.

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+        image="${REGISTRY}/${image}"
+      fi
+    fi
+    # If we're skipping the build, then we just want to upload
+    # the images we already have locally.
+    if [ -z $SKIP_BUILD ]; then
+      archive=${name}
+      echo "Building $image:$TAG ..."
+      $NIX_BUILD --out-link $archive-image -A images.$BUILD_TYPE.$archive --arg allInOne "$ALL_IN_ONE" --arg incremental "$INCREMENTAL" --argstr product_prefix "$PRODUCT_PREFIX"
+      $DOCKER load -i $archive-image
```
Comment (niladrih) [inline scripts/release.sh]: Should it be possible to disable load with a switch?

Comment (tiagolobocastro) [reply scripts/release.sh]: Yes, that's part of the reason that made me want to do this. Will add skopeo support and make loading optional.

Comment (niladrih) [reply scripts/release.sh]: Okay.

**Review by niladrih — APPROVED** [review]
The extensions script, today, has a helm chart script section. Nothing to do here, but an FYI.

**Review by niladrih — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

**Pull request merged**